### ISSUE TITLE
`<li role="presentation">` should not cause `no-invalid-role`

### ISSUE THREAD

**Issue by Techn1x** [ember-template-lint/ember-template-lint#2817]
You can see examples of presentation role being recommended for `li` tags in a few places

MDN docs
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role#example_1_navigation_menu
```html
<ul id="menu" role="menu" aria-labelledby="menubutton">
  <li role="presentation">
    <a role="menuitem" href="#description"> Description </a>
  </li>
  ...
```

Bootstrap docs
https://getbootstrap.com/docs/5.2/components/navs-tabs/#javascript-behavior

```html
<ul class="nav nav-tabs" id="myTab" role="tablist">
  <li class="nav-item" role="presentation">
    <button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home-tab-pane" type="button" role="tab" aria-controls="home-tab-pane" aria-selected="true">Home</button>
  </li>
  ...
```

**Comment by Techn1x** [comment]
Similarly, and perhaps a separate issue - `menuitem` role as below should not violate `require-context-role` - it does not have to be an immediate descendent of group/menu/menubar if the parent role is presentation.

```html
<ul id="menu" role="menu" aria-labelledby="menubutton">
  <li role="presentation">
    <a role="menuitem" href="#description"> Description </a>
  </li>
```

**Comment by bmish** [comment]
Related? #2637

**Comment by Techn1x** [comment]
Yeah looks like that covers the second issue I mentioned in my comment, but not the main issue from the original post

It's definitely all related/adjacent pain points of menus with these rules though
